.github/workflows/cli-ci.yml

```diff
@@ -0,0 +1,69 @@
+name: cli/ubi CI
+
+on:
+  push:
+  pull_request:
+
+jobs:
+  cli-ci:
+    strategy:
+      fail-fast: false
+      matrix:
+        runs-on: [ubicloud, ubicloud-arm]
+    name: cli/ubi CI - ${{matrix.runs-on}}
+    runs-on: ${{matrix.runs-on}}
+
+    env:
+      DB_USER: clover
+      DB_PASSWORD: nonempty
+      DB_NAME: clover_test
+
+    services:
+      postgres:
+        image: postgres:15.4
+        env:
+          POSTGRES_USER: ${{ env.DB_USER }}
+          POSTGRES_PASSWORD: ${{ env.DB_PASSWORD }}
+          POSTGRES_DB: ${{ env.DB_NAME }}
+        ports:
+          - 5432:5432
+        options: --health-cmd pg_isready --health-interval 10s --health-timeout 5s --health-retries 5
+
+    steps:
+    - name: Perform superuser-only actions, then remove superuser
+      run: |
+        psql "postgres://${{ env.DB_USER }}:${{ env.DB_PASSWORD }}@localhost:5432/${{ env.DB_NAME }}" \
+          -c "CREATE EXTENSION citext; CREATE EXTENSION btree_gist; CREATE ROLE clover_password PASSWORD '${{ env.DB_PASSWORD }}' LOGIN; ALTER ROLE ${{ env.DB_USER }} NOSUPERUSER"
+
+    - name: Check out code
+      uses: actions/checkout@v4
+
+    - name: Set up Go
+      uses: actions/setup-go@v5
+      with:
+        go-version: '1.22.2'
```

Comment:
You may, at least eventually, want to have this key off the go.mod files (and then manage the version via asdf in .tool-versions). I think that level of indirection is your best bet (rather than referencing .tool-versions directly), but I'm not sure. It appears to be what we do. Here is the relevant readme bit: https://github.com/actions/setup-go?tab=readme-ov-file#getting-go-version-from-the-gomod-file

I suspect the DEVELOPERS.md file might be ready for some love to describe some of the new setup and stuff at some point also.

Reply:
OK, I switched from `go-version` to `go-version-file`.

I can work on DEVELOPERS.md changes in the near future, when I work on general documentation of the cli.